windows/Dockerfile

```diff
@@ -31,7 +31,7 @@ ENV VS2017BUILDTOOLS_VERSION "16.8.3.0"
 ENV VS2017BUILDTOOLS_SHA256 "ccfb9355f4f753315455542f966025f96de734292d3908c8c3717e9685b709f0"
 #ENV VS2017BUILDTOOLS_DOWNLOAD_URL "https://download.visualstudio.microsoft.com/download/pr/d0eac510-174b-4241-b73b-93dc7cc1fbf7/9822b4c851e14d9658babd1533f66f518c6169196e985fe5713b2774128832ae/vs_BuildTools.exe"
 ENV VS2017BUILDTOOLS_DOWNLOAD_URL "https://download.visualstudio.microsoft.com/download/pr/9b3476ff-6d0a-4ff8-956d-270147f21cd4/ccfb9355f4f753315455542f966025f96de734292d3908c8c3717e9685b709f0/vs_BuildTools.exe"
-ENV GO_VERSION "1.18.5"
+ENV GO_VERSION "1.18.6"
 ENV GO_SHA256 "73753620602D4B4469770040C53DB55E5DD6AF2AD07ECC18F71F164C3224EAAD"
```

Comment:
You probably need to update this

Reply:
Okay! I am not sure what TOFU means as an acronym but yeah, makes sense to take the value from the logs once the build completes

Reply:
Sorry, it's [Trust On First Use](https://en.wikipedia.org/wiki/Trust_on_first_use).

Reply:
No worries, thanks for the link :)